fulfillment-provider/open-api.json

```diff
@@ -0,0 +1,3002 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Order API",
+    "description": "This API specification describes the canonical ShipEngine eCommerce API Order Source module. A module which implements this specification can be integrated with the ShipEngine eCommerce Platform as an Order Source.",
```

Comment:
This text seems wrong - it is referring to Order Soruces.

Reply:
In fact, this entire API spec seems to be describing order sources

Reply:
@joshuaflanagan this is a copy paste error. this file isnt actually used by anything. i dont even know why it exists in order source.